app/src/main/java/com/asfoundation/wallet/ui/gamification/SharedPreferencesGamificationLocalData.kt

```diff
@@ -25,6 +27,15 @@ class SharedPreferencesGamificationLocalData(private val preferences: SharedPref
     }
   }
 
+  override fun setGamificationLevel(gamificationLevel: Int): Completable {
```

Comment:
getLastShownLevel this method contains the current information regarding the last shown level to the user, should we not use this method?

Reply:
Same as above